app/controllers/admin/pages_controller.rb

```diff
@@ -0,0 +1,51 @@
+module Admin
+  class PagesController < Admin::BaseController
+    before_action :set_page, only: [:show, :edit, :destroy]
```

Comment:
__[RuboCop]__

Rails/LexicallyScopedActionFilter: `show` is not explicitly defined on the controller.


<!-- Comment by [SideCI](https://sideci.com) -->